### ISSUE TITLE
Notification player should have black text on white background if thumbnail is not shown

### ISSUE THREAD

**Issue by SameenAhnaf** [TeamNewPipe/NewPipe#6080]
### Checklist

- [x] I am using the latest version - 0.21.1 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Notification player lacks contrast between title bar and player background if thumbnail is not shown. It's quite hard to differentiate between title and player.

### Actual behaviour

Notification player seems to have black text on ash background. This makes title and buttons really hard to see.

### Expected behavior

Notification player should be black and white for better contrast. Personally, I prefer black text with black background. It's quite clean.

### Screenshots/Screen recordings

![Screenshot_2021-04-14-12-20-42-344_org polymorphicshade newpipe](https://user-images.githubusercontent.com/79245641/114763786-02e97e80-9d85-11eb-8954-6bea2cfc28e5.jpg)

### Device info

 - Android version/Custom ROM version:
 - Device model:

**Comment by mhmdanas** [comment]
What should we do if one half of the thumbnail is white and the other is black?

**Comment by SameenAhnaf** [comment]
>Personally, I prefer black text with white background. It's quite clean.

@mhmdanas Btw, my issue is about if "Show thumbnail" is turned off in "Notification" settings. However, this solution could be applicable for black and white thumbnails as you asked for. Daylight theme is expected to be added if someone prefers dark mode.

**Comment by mhmdanas** [comment]
> Btw, my issue is about if "Show thumbnail" is turned off in "Notification" settings.

Oh, yeah, I skimmed quickly through the issue so I didn't notice that part.

**Comment by TiA4f8R** [comment]
Note that the color `#222222` is always set to the notification, [see here](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/java/org/schabi/newpipe/player/NotificationUtil.java#L124-L125). Starting with Android 8.0, this color is overridden with Android MediaStyle notifications if a thumbnail is set to the notification with the generated color by Android. However, if you remove the `#222222` color or a set a transparent color, this will result in a black seekbar on EMUI 10 devices, with no durations shown. See ryanheise/audio_service#289 and https://code.videolan.org/videolan/vlc-android/-/issues/1552 for more details. So if the thumbnail is disabled on the player notification, the option to colorize the notification should be disabled and cannot be enabled to prevent the bug above (which isn't an Android standard, thanks to Huawei and their customizations of Android 10).

**Comment by SameenAhnaf** [comment]
@TiA4f8R Isn't it possible to force black text on white background? I wonder who needs a transparent background.

**Comment by TiA4f8R** [comment]
No, it's something which is managed by Android. In your case, it seems that Xiaomi didn't follow the Android original design, here is my behavior on my device (when I disable the notification without disabling the colorize notification option), an Honor 9X running EMUI 10.0.0:

![Screenshot with seekbar](https://user-images.githubusercontent.com/74829229/114891156-ee03fe00-9e0b-11eb-8eba-afa1d1b4dc76.jpg)


![Screenshot without seekbar](https://user-images.githubusercontent.com/74829229/114891170-f1978500-9e0b-11eb-81eb-ec9c7340393f.jpg)

**Comment by SameenAhnaf** [comment]
@TiA4f8R What about a separate setting? Enabling "Force White Thumbnail" should show black text on white thumbnail. Note could be like "Enable this if disabling "Show thumbnail" misbehaves".

**Comment by SameenAhnaf** [comment]
Guys, it seems like that white text or black thumbnail is enforced on the notification player if "Load thumbnails" option is disabled regardless of theme.

However, showing thumbnail within the app and the notification aren't the same thing. Maybe, we need a separate setting: "Force Analogue Thumbnail", subnote: "Enable this if disabling "Show thumbnail" misbehaves on your device"

If white theme is enabled: Black text on white background
If dark theme is enabled: White text on grey background
If white theme is enabled: White text on black background (or thumbnail)


https://user-images.githubusercontent.com/79245641/116861572-19b51f80-ac25-11eb-9b6c-d23ffac6e30e.mp4
